css-three-column-layout/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="header">
+      <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+    </div>
+    <div class="row img-row">
+      <div class="column-one-third img-column">
+        <img src="images/merge.png" alt="Streamlined QMS" class="icon-image">
+      </div>
+      <div class="column-one-third img-column">
+        <img src="images/cloud-computing.png" alt="Integrates with existing tools" class="icon-image">
+      </div>
+      <div class="column-one-third img-column">
+        <img src="images/discussion.png" alt="In-depth Onboarding" class="icon-image">
+      </div>
+    </div>
+    <div class="row text-row">
+      <div class="column-one-third text-column">
+        <h3>Streamlined QMS</h3>
+        <p>A QMS that includes modules for all stages of product life cycle from Design Control to CAPA</p>
+      </div>
+      <div class="column-one-third text-column">
+        <h3>Integrates with existing tools</h3>
+        <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third text-column">
+        <h3>In-depth Onboarding</h3>
+        <p>We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```

Comment:
I can understand why you would do this as related to layout classes, but this sort of HTML setup makes it look as though the image is separate content from the text for each column, and that is not an accurate representation of the content. The icon, heading, and descriptive text should all be grouped together in one `column` class. Remember, the goal of HTML is to organize your elements in a way that semantically describes the content accurately. If elements are related, they should be grouped together so it's clear through the HTML that those items are related content.